qualtran/resource_counting/_bloq_counts.py

```diff
@@ -152,8 +193,10 @@ def __mul__(self, other):
             cswap=other * self.cswap,
             and_bloq=other * self.and_bloq,
             clifford=other * self.clifford,
-            rotation=other * self.rotation,
             measurement=other * self.measurement,
+            binned_rotation_epsilons=Counter(
+                {eps_bin: other * n_rot for eps_bin, n_rot in self.binned_rotation_epsilons.items()}
+            ),
```

Comment:
so how does it work when adding or multiplying when there are values that differ only in precision

Reply:
the `__mul__` is to multiply with some int `other`, right? which just multilpies the frequencies, not affecting the `epsilons`.

For add, it just combines the two counters together, and equal keys will get merged. (handled by `Counter.__add__`). The precision of representing epsilon is only used when converting it to a string key in `from_rotation_with_eps`, but not stored in the GateCounts object itself.